dev/service-registry/play-integration-javadsl/src/main/scala/com/lightbend/lagom/play/LagomPlayModule.scala

```diff
@@ -23,7 +23,7 @@ class LagomPlayModule extends Module {
   private val logger = Logger(this.getClass)
 
   override def bindings(environment: Environment, config: Configuration): Seq[Binding[_]] = {
-    val maybeServiceInfoBinding: Option[Binding[ServiceInfo]] = prepareServiceInfoBinding(config)
+    val maybeServiceInfoBinding: Option[Binding[ServiceInfo]] = prepareServiceInfoBinding(config.underlying)
```

Comment:
In this place I'm keeping the Play Configuration because this is an extension of `play.api.inject.Module`